Lib/csv.py

```diff
@@ -130,17 +130,14 @@ def __next__(self):
         return d
 
     __class_getitem__ = classmethod(types.GenericAlias)
-
-
```

Comment:
Would you mind not removing existing empty lines like this one, and not adding many new empty lines?
They are used to group sections of code visually.

Reply:
oh ! sorry ! i'll correct it.